### PR TITLE
feature: dark mode for welcome site

### PR DESCRIPTION
**Pull request by aakankshadhurandhar**
https://user-images.githubusercontent.com/34390765/161393101-3b88526b-cd45-4d8f-95d5-c14bc806dc18.mp4

https://user-images.githubusercontent.com/34390765/161393667-0dc145ba-f879-4167-9209-8e154324750b.mp4

### CODE REVIEW

File: js/darkmode.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    document.body.classList.add('dark-theme');
+    icon.src = 'img/icons/sun.png';
+  }
+}
+
+function setCookie(name, value, days = 30) {
+  const domain = '.realdevsquad.com';
+  const date = new Date();
+  date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000);
+  const expires = '; expires=' + date.toUTCString();
```
Comment (akshay1502) [inline js/darkmode.js:29]: do we need `; expires` here?

File: index.html [hunk 2]
```diff
@@ -90,8 +90,19 @@
                 Status
               </a>
             </li>
+
+            <li>
+              <img
+                id="icon"
```
Comment (akshay1502) [inline index.html:96]: Lets have `themeLogo` as meaningful id name

File: js/darkmode.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    icon.src = 'img/icons/sun.png';
+    setCookie('theme', 'dark', 30);
+  } else {
+    icon.src = 'img/icons/moon.png';
+    setCookie('theme', 'light', 30);
+  }
+};
+
+function onload() {
+  if (getCookie('theme') === 'light') {
```
Comment (akshay1502) [inline js/darkmode.js:16]: What if the cookie is not present before? It will show dark mode!!🤔

**Comment by akshay1502** [comment]
Make discord.html, faq.html, and code-of-conduct.html pages into dark mode too.

File: js/darkmode.js (hunk 3 above)
Comment (aakankshadhurandhar) [reply js/darkmode.js:16]: Yes It is showing dark mode as default . Shall I change it to light Mode as default ?

Comment (akshay1502) [reply js/darkmode.js:16]: Yes, light mode is default.

File: index.html (hunk 2 above)
Comment (aakankshadhurandhar) [reply index.html:96]: done

File: js/darkmode.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  } else {
+    setCookie('theme', 'light', 30);
+  }
+});
+
+function setCookie(name, value, days = 30) {
+  const domain = '.realdevsquad.com';
+  const date = new Date();
+  date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000);
+  const expires = '; expires=' + date.toUTCString();
```
Comment (rohan09-raj) [inline js/darkmode.js:17]: We can remove `; expires=` string as we are already adding it in line 18

**Review by rohan09-raj — CHANGES_REQUESTED** [review]
We are left with `Code of Conduct` page to be in Dark-Mode
https://deploy-preview-227--staging-welcome.netlify.app/code-of-conduct.html
Also, please make the background of nav-menu in mobile-view as `E1E1EC`
Please make these changes, then I guess we can merge this.

**Review by rohan09-raj — APPROVED** [review]
Nicely Done 💯 💯, thanks for constantly asking for reviews. This can be merged I think now

**Pull request merged**